src/app/models/field/dateField.ts

```diff
@@ -23,6 +23,10 @@ const createDateFieldSchema = () => {
         default: null,
       },
     },
+    invalidDaysOfTheWeek: {
+      type: [Number],
```

Comment:
do we want to use the DaysOfWeek enum instead? Allows for better type and validation

Reply:
Updated PR